vgrep/PKGBUILD

```diff
@@ -1,35 +1,39 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
 # Contributor: Darshit Shah <darnir@gmail.com>
+# Contributor: Jonas Malaco <jonas@protocubo.io>
 
 pkgname=vgrep
 pkgver=2.4.0
 pkgrel=1
 pkgdesc="Reimpementation of the ancient cgvg perl scripts"
-arch=('x86_64' 'i686')
```

Comment:
Technically this is not an official Arch Linux package but an AUR Arch Linux package. There might be 32bit users who are using this PKGBUILD say through ArchLinuxArm. We should keep that support. If this was an official Arch Linux package I agree.

Reply:
My bad, I considered ArchLinuxArm but assumed it would rerquire ARM-specific values, and thus wasn't supported with the current tags.  Now I see that many packages there don't list any ARM-specific architectures.

Also, I forgot about ArchLinux32... I'll just revert this.